scipy/conftest.py

```diff
@@ -153,6 +153,12 @@ def check_fpu_mode(request):
     except ImportError:
         pass
 
+    try:
+        import dask.array  # type: ignore[import-not-found]
+        xp_available_backends.update({'dask.array': dask.array})
```

Comment:
```suggestion
        import array_api_compat.dask.array as da  # type: ignore[import-not-found]
        xp_available_backends.update({'dask.array': da})
```

This is the change that I'm talking about that fixes a lot of things.

Assuming making this change looksgood, I'll cleanup previous unneeded fixes locally and send in another PR to add this + fix merge conflicts.

Reply:
We don't want to do this, actually. We want to pass arrays from the unwrapped namespace in tests to check that we correctly coerce with `array_namespace`. If you need to use standard functions in tests, you can write `xp_test = array_namespace(x)`. It's good to add a comment about what's missing from the unwrapped namespace.

Reply:
Got it, I'll try to add individual ``array_namespace`` calls and report back on what the diff looks like after.

Reply:
If you grep for `xp_test` you'll see examples of the pattern. There is another pattern like `acos = array_namespace(x).acos` - either is fine.